frontend/src/components/Card.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+
+function Card({ cocktailName, image }) {
+  return (
+    <div className="card">
```

Comment:
This element might be a figure instead of a div. There is an img with a text. Take a look at html element figure.